src/frontend/src/lib/i18n/en.json

```diff
@@ -19,7 +19,8 @@
 			"test_banner": "For testing purposes only!"
 		},
 		"alt": {
-			"logo": "$name logo"
+			"logo": "$name logo",
+			"background": "$oisy_name background"
```

Comment:
Not an a11y expert but maybe the description can be a bit more descriptive?

Reply:
For example yes thanks

Reply:
done!